### ISSUE TITLE
Plugin logging levels cannot be set

### ISSUE THREAD

**Issue by bgd42** [yamanyar/restrict-maven-plugin#9]
Hi,

Seems that in `RestrictLogger` the passed in proper `Log` is not being used. Instead the plugin is using `SystemStreamLog` which is dumping everything with `System.out.print*()`.

This has the effect that we get massive amounts of [debug] statements, without being able to disable it:
```
[info] Restriction report path is:C:\dev\snapshots\repair\controller\cloud-integration-api\target\restrict-maven-plugin.txt
[info] restrict-maven-plugin started!
[info] ###### Restriction Rules (begins) ######
[info] [1-2] Access from com.xxx.* (except com.xxx.TestA,com.xxx.*Test) to com.xxx.TestA (except com.xxx.TestA,com.xxx.*Test) will be not allowed.
[info] ###### Restriction Rules (ends)   ######
[debug] Inspecting org.slf4j:slf4j-api:jar:1.7.12:compile
[debug] Inspecting org.glassfish.jersey.core:jersey-client:jar:2.18:compile
[debug] Inspecting javax.ws.rs:javax.ws.rs-api:jar:2.0.1:compile
(...)
[debug] An exception to a restriction (com.xxx.* of rule 1) matched com.xxx.monitor.TemporalConsumerTest.
[debug] An exception to a restriction (com.xxx.* of rule 1) matched com.xxx.QualifiedNameMatcherTest.
[debug] An exception to a restriction (com.xxx.* of rule 1) matched com.xxx.QualifiedNameTest.
(...)
```

Any plans for fixing this?

Thank you,
Bogdan

**Comment by yamanyar** [comment]
Hi,

Yes; I will fix that in 0.8 release

**Comment by yamanyar** [comment]
Hi,

This logger uses the default logger but also create a special file to write down all the restrict faults. So there are not much things to achieve.

However in 0.8; I added printDebugs property stop printing debugs.
